frontend/scripts/update-api-client.sh

```diff
@@ -5,7 +5,7 @@ rm -f packages/api_client/apis/*.ts
 rm -f packages/api_client/models/*.ts
 
 docker run --rm --network="host" -v ${PWD}:/local \
-    openapitools/openapi-generator-cli generate \
+    openapitools/openapi-generator-cli:v4.3.1 generate \
```

Comment:
I've added the version tag here so that there won't be unnecessary format changes, and we can investigate new versions if we need to.

Reply:
Nice fix!